anchor/common/qbft/src/lib.rs

```diff
@@ -905,22 +908,29 @@ where
         data_hash: D::Hash,
         round_change_justification: Vec<SignedSSVMessage>,
         prepare_justification: Vec<SignedSSVMessage>,
+        round: Option<Round>,
```

Comment:
This was implemented for the spec tests by @Zacholme7 I think. Not sure if there's a cleaner way to handle this or if this was a hack or permanent change. Maybe we want to clarify